hudi-sync/hudi-datahub-sync/src/main/java/org/apache/hudi/sync/datahub/DataHubSyncTool.java

```diff
@@ -83,20 +83,20 @@ public void syncHoodieTable() {
     }
   }
 
-  private void syncSchema() throws Exception {
```

Comment:
Just curious why are we removing throws Exception? Is it because you want to enforce more stricter exception handling in the caller?